app/models/sponsorship.rb

```diff
@@ -30,23 +30,12 @@ class Sponsorship < ActiveRecord::Base
 
 private
 
-  def start_date_no_later_than_1st_of_next_month
-    first_of_next_month = Date.current.beginning_of_month.next_month
-    if (self.start_date > first_of_next_month)
-      errors[:start_date] << "can not be later than the first of next month"
-    end
-  end
-
```

Comment:
@b264 do you remember the utility of this private method ? I didn't find any references through the source code :thinking: